src/windows/editor_object_selection.c

```diff
@@ -2116,19 +2169,22 @@ static bool filter_chunks(rct_object_entry *entry, rct_object_filters *filter)
 static void filter_update_counts()
 {
 	if (!_FILTER_ALL || strlen(_filter_string) > 0) {
+		int numObjects = gInstalledObjectsCount;
```

Comment:
I also kinda copied numObjects from visible_list_refresh() (i was using that when referencing if it was okay to reverse the for loop in filter_update_counts() (as originally the for decremented the number), should i remove it from visible_list_refresh() too?